app/routes/meetings/components/MeetingEditor.tsx

```diff
@@ -183,6 +184,23 @@ const MeetingEditor = ({
                 component={TextArea.Field}
               />
               <div className={styles.sideBySideBoxes}>
+                <FormSpy
+                  subscription={{ values: true }}
+                  onChange={({ values }) => {
+                    const startTime = moment(values.startTime);
+                    const endTime = moment(values.endTime);
+                    if (endTime.isBefore(startTime)) {
+                      form.change(
+                        'endTime',
+                        startTime
+                          .clone()
+                          .add(1, 'hour')
+                          .add(45, 'minutes')
```

Comment:
Any specific reason for choosing 1 hour and 45 minutes and not just 2 hours or something?

Reply:
Because a lot of student meetings last 1 hour and 45 minutes :)

Reply:
Sure, with akademisk kvarter ™️ that makes sense